app/javascript/styles/_colors.scss

```diff
@@ -9,3 +9,4 @@ $warning-icon-color: #C45500;
 
 $danger-color: rgb(219, 50, 20);
 $success-color: #75b041;
```

Comment:
TOL:  The existing colors defined here `danger` and `success` are used pretty universally across the app, but I'm not sure `options` has the same general application. It seems we don't have a strong habit of adding colors to this file but considering it's titled `colors.scss` it makes sense to add here haha and I'm wondering if we ought to do more of that.. would it make sense to title this `options-background-color` though to be most specific?